_typos.toml

```diff
@@ -0,0 +1,14 @@
+[default.extend-words]
+# Don't correct the word
+
+# test for incorrect contract_address key in deserialize_get_storage_input
+addresss = "addresss"
+
+# test for incorrect block_number key in deserialize_block_id_block_number_variants
+numberr = "numberr"
+
+# STARKNET_1.0_MSGING_L1TOL2_MAPPPING_V2
+MAPPPING = "MAPPPING"
+
+payed = "payed"
+ede = "ede"
```

Comment:
Ok I may have found the answer in the [docs](https://crates.io/crates/typos) under `False positives`

Reply:
Perhaps the entire submodule can be ignored, to prevent false negatives (i.e. somebody writing `payed` outside of the submodule and it not being registered as an error)

Reply:
Ok, it's more reasonable.
1. I will ignore the entire directory "contracts/l1-l2-messaging/solidity/" instead just ignore `MAPPPING` and `payed`. I think it's a submodule?
2. `addressss` and `numberr` are constructed as incorrect keys on purpose for test cases in `crates/starknet-devnet-server/src/api/json_rpc/models.rs`. Maybe it's better to just ignore this file.